src/Fist/Http/src/Request.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace Fist\Http;
+
+class Request
+{
+    protected $server;
+
+    protected $baseUrl;
+
+    protected $path;
+
+    protected $requestUri;
+
+    public function __construct(array $query = [], array $request = [], array $cookies = [], array $files = [], array $server = [], $content = null)
+    {
+        $this->initialize($query, $request, $cookies, $files, $server, $content);
+    }
+
+    public static function createFromGlobals()
```

Comment:
__[phpmd]__
* createFromGlobals accesses the super-global variable $_GET.
* createFromGlobals accesses the super-global variable $_POST.
* createFromGlobals accesses the super-global variable $_COOKIE.
* createFromGlobals accesses the super-global variable $_FILES.
* createFromGlobals accesses the super-global variable $_SERVER.

<!-- Comment by [SideCI](https://sideci.com) -->